src/test/java/duke/parse/ParserTest.java

```diff
@@ -0,0 +1,182 @@
+package duke.parse;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.fail;
+
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.time.LocalTime;
+
+import org.junit.jupiter.api.Test;
+
+import duke.parse.command.AddCommand;
+import duke.parse.command.EchoCommand;
+import duke.parse.command.EmptyCommand;
+import duke.parse.command.ExitCommand;
+import duke.parse.command.ListCommand;
+import duke.parse.command.MarkCommand;
+import duke.parse.command.SaveCommand;
+import duke.task.Deadline;
+import duke.task.Event;
+import duke.task.ToDo;
+
+public class ParserTest {
+    private void notifyError() {
+        fail("should not throw any error");
+    }
+
+    @Test
+    public void emptyTest() {
+        try {
+            assertEquals(new EmptyCommand(), Parser.parse(""));
+        } catch (Parser.ParseError e) {
+            notifyError();
+        }
+    }
```

Comment:
Perhaps would it be better if we renamed the test from emptyTest to the format "featureUnderTest_testScenario_expectedBehavior()"?